cloud/backend/cf-d1/cf-hono-server.ts

```
@@ -0,0 +1,347 @@
+  readonly _onClose: () => void;
+  constructor(
+    onClose: () => void = () => {
+      /* */
+    },
+  ) {
+    this._onClose = onClose;
+  }
+  async inject(
+    c: Context,
+    // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
+    fn: (rt: ExposeCtxItemWithImpl<CFWSRoom>) => Promise<Response | void>,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)